frontend/src/Components/pages/dashboard/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import NavigationBar from "../../navigationbar";
+import { Grid } from "@material-ui/core";
+
+const Dashboard = () => {
+  return (
+    <Grid container direction="column" justify="space-between" alignItems="stretch" spacing={2}>
+      <Grid item>
+        <NavigationBar />
```

Comment:
We should probably shift the nav bar to the App.js level so it doesnt render a new one for each page

Reply:
This and the other two comments should be resolved in this commit https://github.com/junwei26/remiNdUS/pull/1/commits/fd0890f238843b571d600f8ca589655a002668e5
This is the first commit right after this PR.